tests/test_gates.py

```diff
@@ -52,6 +53,7 @@ def op(op_name):
         "ControlledPhaseShift": [qml.ControlledPhaseShift, [], {"phi": 1.777, "wires": [0, 1]}],
         "CPhase": [qml.CPhase, [], {"phi": 1.777, "wires": [0, 1]}],
         "MultiRZ": [qml.MultiRZ, [], {"theta": 0.112, "wires": [0, 1, 2]}],
+        "GlobalPhase": [qml.GlobalPhase, [], {"phi": 0.112, "wires": [0, 1, 2]}],
```

Comment:
Should we add an out of order wire test too?

Reply:
I could, but then should we do it for all the other gates? This is probably the least sensitive gate since the `wires` argument is actually unused in the C++ kernels.

Reply:
Let's look at this in a follow-up improvement of the test suite.